domain/macaroon/state/rootkeys_test.go

```diff
@@ -23,38 +23,40 @@ type rootKeyStateSuite struct {
 var _ = gc.Suite(&rootKeyStateSuite{})
 
 var (
+	now = time.Now()
```

Comment:
We need to use a unique time across the tests, otherwise, it breaks because of time falling over a second boundary.